File: app/src/main/java/cu/axel/smartdock/fragments/KeyboardPreferences.java
package cu.axel.smartdock.fragments;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import cu.axel.smartdock.R;

public class KeyboardPreferences extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle arg0, String arg1) {
        setPreferencesFromResource(R.xml.preferences_keyboard, arg1);
    }

}

File: app/src/main/java/cu/axel/smartdock/fragments/KeyboardPreferences.kt
package cu.axel.smartdock.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import cu.axel.smartdock.R

class KeyboardPreferences : PreferenceFragmentCompat() {
    override fun onCreatePreferences(arg0: Bundle?, arg1: String?) {
        setPreferencesFromResource(R.xml.preferences_keyboard, arg1)
    }
}
